src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+import * as React from 'react';
+
+import * as Data from '../data';
+
+const fetchSettings: RequestInit = {
+    credentials: "same-origin",
+    headers: {
+        ["X-Requested-With"]: "XMLHttpRequest"
+    },
+};
+
+class LoadingPanel extends React.Component<undefined, undefined> {
+    render() {
+        return <div className="progress">
+            <div className="progress-bar progress-bar-striped active" style={{ width: "100%" }}>
+                <span className="sr-only">Loading...</span>
+            </div>
+        </div>
+    }
+};
+
+class ErrorPanel extends React.Component<{ message: string }, undefined> {
+    render() {
+        return <div className="alert alert-danger">{this.props.message}</div>;
+    }
+};
+
+class UserIcon extends React.Component<{ user: Data.User }, undefined> {
+    render() {
+        return <a href={this.props.user.url}>
+            <img style={{ "margin": "0.1em" }} height="32" width="32" src={this.props.user.avatarUrl} title={this.props.user.login} alt={this.props.user.login} />
+        </a>;
+    }
+}
+
+class IssueUserView extends React.Component<{ issue: Data.Issue }, undefined> {
+    render() {
+        let issue = this.props.issue;
```

Comment:
Another minor point, but it's more conventional to use `const` rather than `let` by default, and only use `let` for the special cases where you need to emphasise that the variable's will get written to after definition.